bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -571,6 +583,154 @@ public KNXCoreTypeMapper() {
         defaultDptMap.put(HSBType.class, DPTXlatorRGB.DPT_RGB.getID());
     }
 
+    /*
+     * This function computes the target unit for type conversion from OH quantity type to DPT types.
+     * Calimero library provides units which can be used for most of the DPTs. There are some deviations
+     * from the OH unit scheme which are handled.
+     */
+    private String quantityTypeToDPTValue(QuantityType<?> qt, int mainNumber, int subNumber, String targetOhUnit)
+            throws KNXException {
+        double scaleFactor = 1.0;
+        switch (mainNumber) {
+            case 7:
+                switch (subNumber) {
+                    case 3:
+                    case 4:
+                        targetOhUnit = "ms";
+                        break;
+                }
+                break;
+            case 9:
+                switch (subNumber) {
+                    // special case: temperature deltas specified in different units
+                    // ignore the offset, but run a conversion to handle prefixes like mK
+                    // scaleFactor is needed to properly handle °F
+                    case 2: {
+                        final String unit = qt.getUnit().toString();
+                        // find out if the unit is based on °C or K, getSystemUnit() does not help here as it always
+                        // gives "K"
+                        if (unit.contains("°C")) {
+                            targetOhUnit = "°C";
+                        } else if (unit.contains("°F")) {
+                            targetOhUnit = "°F";
+                            scaleFactor = 5.0 / 9.0;
+                        } else if (unit.contains("K")) {
+                            targetOhUnit = "K";
+                        } else {
+                            targetOhUnit = "";
+                        }
+                    }
+                        break;
+                    case 3: {
+                        final String unit = qt.getUnit().toString();
+                        if (unit.contains("°C")) {
+                            targetOhUnit = "°C/h";
+                        } else if (unit.contains("°F")) {
+                            targetOhUnit = "°F/h";
+                            scaleFactor = 5.0 / 9.0;
+                        } else if (unit.contains("K")) {
+                            targetOhUnit = "K/h";
+                        } else {
+                            targetOhUnit = "";
+                        }
+                    }
+                        break;
+                    case 23: {
+                        final String unit = qt.getUnit().toString();
+                        if (unit.contains("°C")) {
+                            targetOhUnit = "°C/%";
```

Comment:
What is this unit ?

Reply:
9.023 DPT_KelvinPerPercent, this is a delta and we need to convert to the matching base unit first (automatic conversion °C to K or °F will fail due to the offset/scaling)

Reply:
In fact, my question was: is "°C/%" is a valid unit supported by UoM ?

Reply:
Actually is seem so. DSL rules let you construct this
```
        var k = 25|K
        var p = 10|%
        var result = k/p
        var dimensionless = p.toUnit("")
```
Printing to the log gets you
```
...script.UoM   ] - this is k: 25 K
...script.UoM   ] - this is p: 10 %
...script.UoM   ] - this is k/p: 2.5 K/%
...script.UoM   ] - this is p converted to dimensionless: 0.1
```
The values are sent to the KNX bus when I use sendCommand, both tested with items defined as Number and Number:Temperature. It leads to same value on KNX side.

Reply:
Ok, strange but if you tested it, I believe you.